packages/e2e-tests/specs/editor/plugins/allowed-blocks.test.js

```diff
@@ -29,7 +29,6 @@ describe( 'Allowed Blocks Filter', () => {
 			await page.$x( `//button//span[contains(text(), 'Paragraph')]` )
 		 )[ 0 ];
 		expect( paragraphBlockButton ).not.toBeNull();
-		await paragraphBlockButton.click();
```

Comment:
I think this was creating a race condition with the searchBlock call right after. One of them was opening the inserter while the other was closing it.